<ui version="4.0">
 <class>AddEditCoffeeForm</class>
 <widget class="QDialog" name="AddEditCoffeeForm">
  <property name="geometry">
   <rect>
    <x>100</x>
    <y>100</y>
    <width>300</width>
    <height>300</height>
   </rect>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="nameEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>280</width>
      <height>30</height>
     </rect>
    </property>
           <property name="placeholderText">
        <string>Введите сорт</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="roastEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>280</width>
      <height>30</height>
     </rect>
    </property>
           <property name="placeholderText">
        <string>Введите Степень прожарки</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="groundEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>280</width>
      <height>30</height>
     </rect>
    </property>
           <property name="placeholderText">
        <string>В зернах или молотый?</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="descEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>280</width>
      <height>30</height>
     </rect>
    </property>
           <property name="placeholderText">
        <string>Введите описание</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="priceEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>170</y>
      <width>280</width>
      <height>30</height>
     </rect>
    </property>
           <property name="placeholderText">
        <string>Введите цену</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="volumeEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>210</y>
      <width>280</width>
      <height>30</height>
     </rect>
    </property>
           <property name="placeholderText">
        <string>Введите объем</string>
    </property>
   </widget>
   <widget class="QPushButton" name="okButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>250</y>
      <width>280</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>OK</string>
    </property>
   </widget>
  </widget>
 </widget>
</ui>
